rosserial_python/src/rosserial_python/SerialClient.py

```diff
@@ -520,7 +538,8 @@ def run(self):
             except IOError:
                 # One of the read calls had an issue. Just to be safe,
                 # request that the client reinitialize their topics.
-                rospy.logwarn("Requesting reinitialization of client topics!")
+                # rospy.logwarn("Requesting reinitialization of client topics!")
+                time.sleep(0.002)
```

Comment:
What does this delay do?

Reply:
hmmm..... not particularly useful; bit of time for the port to come back up?